### DISCUSSION TITLE
Warm Node full without shards but with data in OpenSearch

### DISCUSSION THREAD

**Discussion by Shadeblad3** [community/community#78672] [Programming Help]
### Select Topic Area

Question

### Body

Hello everyone,

I'm using OpenSearch, and I have a series of warm nodes that are functioning correctly. However, the other day I noticed that one of them is almost full (about to reach the lower watermark), but it turns out it has no shards assigned because I decided to exclude it to check if it was due to the shards it had. It turns out that even without shards, it is filled with 3.7TB out of 4.1TB total.

In the folder /opt/docker_files/els/data/nodes, when I run ls -l --block-size=GB, it returns many indices, each with 1GB, and there are more than 250 indices. Inside each of those folders, there are two additional folders with 1GB each, indicating that the node is somehow full of information from indices even when that information shouldn't be there.

The question here would be: What advice would you give me to free up the node from this information without deleting everything? The straightforward option would be to make a backup and delete the folder. So far, I've tried restarting the Docker container "els," but it hasn't helped.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Shadeblad3 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by Shadeblad3** [comment]
Thank you @queenofcorgis for the help. I hope I can receive an answer sooner or later.
